docs/OTFFT.ipynb

```diff
@@ -0,0 +1,178 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.        x[:] = scipy.fft.fft(x)  # we will implement our fft shortly!
</pre><p>We will later work on implementing this <code>fft</code> function using 6-step / 8-step algorithm. We then change this function accordingly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/939/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>